package.json

```diff
@@ -1,19 +1,30 @@
 {
-    "name": "infusion-docs",
-    "version": "1.5.0",
-    "description": "Infusion Documentation",
-    "license" : "CC-BY-3.0",
-    "repository": "https://github.com/fluid-project/infusion-docs",
-    "scripts": {
-        "docpad": "docpad run --env static",
-        "deploy": "docpad deploy-ghpages --env static"
-    },
-    "dependencies": {
-        "URIjs": "^1.14.1",
-        "docpad-plugin-ghpages": "~2.4.4",
-        "docpad-plugin-handlebars": "~2.3.0",
-        "docpad-plugin-highlightjs": "~2.2.2",
-        "docpad-plugin-marked": "~2.2.1",
-        "fs-extra": "^0.18.0"
-    }
+  "name": "infusion-docs",
+  "version": "1.5.0",
+  "description": "Infusion Documentation",
+  "license": "CC-BY-3.0",
+  "repository": "https://github.com/fluid-project/infusion-docs",
+  "scripts": {
+    "docpad": "docpad run --env static",
+    "deploy": "docpad deploy-ghpages --env static",
+    "pretest": "docpad generate --env static",
+    "test": "node tests/all-tests.js"
+  },
+  "dependencies": {
+    "URIjs": "^1.14.1",
```

Comment:
Given this diff has been generated for the time being, we should take the opportunity to adjust these to our project standard semver usage of only binding to exact versions

Reply:
Absolutely, I only avoided it because I thought there might be some reason for it.

Reply:
Done.